src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String COLOUR = "white";
+    private static final int RADIUS = 10;
+
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int randomFigureSide = random.nextInt(13);
+
+    public Figure getDefaultFigure() {
+        return new Circle(COLOUR, RADIUS);
+    }
+
+    public Figure getRandomFigure() {
+        String randomColor = colorSupplier.getRandomColor();
+        int randomFigureNumber = random.nextInt(6);

```

Comment:
```suggestion
        int randomFigureNumber = random.nextInt(6);

```
Why 6? It’s also a magic number